homework2/task01/project1/task1.py

```diff
@@ -0,0 +1,78 @@
+"""
+Given a file containing text. Complete using only default collections.
+
+1) Find 10 longest words consisting from largest amount of unique symbols
+2) Find rarest symbol for document
+3) Count every punctuation char
+4) Count every non ascii char
+5) Find most common non ascii char for document
+"""
+import re
+from collections import Counter
+from typing import List
+
+
+def file_presettings(file_path: str) -> str:
+    r"""
```

Comment:
You don't need that r in the docstring.

Reply:
It was a mistake in console  : D301 Use r""" if any backslashes in a docstring. But I found how to fix it. I add D301 in .flake file in ignore section.